### ISSUE TITLE
[FEAT] Update JOURNEYS to offer updated rSeries tenant version support (through v15.1.9)

### ISSUE THREAD

**Issue by maranianf5** [f5devcentral/f5-journeys#114]
**Is your feature request related to a problem? Please describe.**
Yes, we currently offer tenant support up to version 15.1.9 for tenants running on rSeries platforms. However, customers using JOURNEYS are only presented with options of 15.1.5 and 15.1.6 for destination device versions. Attached screenshot below from the JOURNEYS GUI.

**Describe the solution you'd like**
Update JOURNEYS to offer destination device version support through 15.1.9 in rSeries, as per https://my.f5.com/manage/s/article/K86001294.

**Describe alternatives you've considered**
None that I can think of as the tool appears to be out of date.

![image](https://github.com/f5devcentral/f5-journeys/assets/73652915/914963a5-4c08-488f-944b-0d4ab5f0a355)

**Comment by mikeoleary** [comment]
Agreed, I think we need an option to export UCS for 15.1.x all the way through the latest. 

Understood that this may require a Journeys Tool update every time there is a 15.1.x BIG-IP release. If there's a workaround we could use documentation on that too.

**Comment by f5-purc** [comment]
Internal tracking [MBIPMP-25462]

**Comment by petrcech** [comment]
Hi,
on a side note. The 17.1.x BIG-IP versions are now also supported on rSeries HW, so it would be nice, if they were also supported.
Thanks
